optimize/apis-tools/optimize-api/tutorial.md

```diff
@@ -0,0 +1,162 @@
+
+1. To set up your credentials, create an `.env` file which will be protected by the `.gitignore` file. These keys will be consumed by the `auth.js` file to execute the OAuth protocol, and should be saved when you generate your client credentials in [prerequisites](#prerequisites).
+2. Examine the existing `.env.example` file for an example of how your `.env` file should look upon completion. You will need to add your `COMPONENTS_CLIENT_ID`, `COMPONENTS_CLIENT_SECRET`, `OPTIMIZE_BASE_URL`, and `OPTIMIZE_AUDIENCE`, which is `https://optimize.camunda.io` in a Camunda 8 SaaS environment.
+
+:::note
+
+In this tutorial, we will execute arguments to list existing dashboard IDs and delete a dashboard. You can examine the framework for processing these arguments in the `cli.js` file before getting started.
+
+:::
+
+## GET a list of existing dashboard IDs
+
+First, let's script an API call to list our existing dashboard IDs.
+
+To do this, take the following steps:
+
+1. Examine the function `async function listDashboards([collectionId])` at the top of the `optimize.js` file. This is where you will script out your API call.
+2. Within the function, you must first apply an access token for this request:
+
+```
+const accessToken = await getAccessToken("components", optimizeAudience);
+```
+
+3. Using your generated client credentials from [prerequisites](#prerequisites), call your Optimize audience and base URL. Ensure these credentials are added to your `.env` file, and script the references to the values. For example:
+
+```
+const optimizeAudience = process.env.OPTIMIZE_AUDIENCE;
+const optimizeApiUrl = process.env.OPTIMIZE_BASE_URL;
+```
+
+4. Script the API endpoint to list the dashboard IDs of a particular collection:
+
+```
+const url = `${optimizeApiUrl}/api/public/dashboard?collectionId=${collectionId}`;
+```
+
+5. Configure your GET request to the appropriate endpoint, including an authorization header based on the previously acquired `accessToken`:
+
+```
+  const options = {
+    method: "GET",
+    url,
+    headers: {
+      Accept: "application/json",
+      Authorization: `Bearer ${accessToken}`
+    }
+  };
+```
+
+6. Call the collection's endpoint, process the results from the API call, emit the dashboard IDs to output, and emit an error message from the server if necessary:
+
+```
+  try {
+    const response = await axios(options);
+    const results = response.data;
+
+    results.forEach(x => console.log(`ID: ${x.id}`));
+  } catch (error) {
+    // Emit an error from the server.
+    console.error(error.message);
+  }
+```
+
+7. In your terminal, run `node cli.js optimize list` for a list of your existing dashboard IDs.
```

Comment:
non-blocking question: 

Should we show example output here?

Reply:
Added in latest commit 👍